src/averageAges.js

```diff
@@ -15,11 +16,13 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter(person =>
+    (!century || Math.ceil(person.died / 100) === century) && person.sex === 'm'
+  );
+
+  const totalAge = men.reduce((sum, person) => sum + (person.died - person.born), 0);
```

Comment:
You can move this condition in separate function, because u use the same in calculateWomenAverageAge
![image](https://github.com/mate-academy/js_average-ages/assets/54936044/eed1dead-20b7-4ff8-a5a5-37b66e6ae49d)